etherman/feijoa_contracts.go

```diff
@@ -0,0 +1,38 @@
+package etherman
+
+import (
+	"github.com/0xPolygonHermez/zkevm-node/etherman/smartcontracts/feijoapolygonzkevm"
+	"github.com/0xPolygonHermez/zkevm-node/log"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// FeijoaContracts represents the contracts of the Feijoa upgrade
+type FeijoaContracts struct {
+	FeijoaZKEVMAddress common.Address
+	FeijoaZKEVM        *feijoapolygonzkevm.Feijoapolygonzkevm
+	//FeijoaRollupManager         *feijoapolygonrollupmanager.Feijoapolygonrollupmanager
+	//FeijoaGlobalExitRootManager *feijoapolygonzkevmglobalexitroot.Feijoapolygonzkevmglobalexitroot
```

Comment:
What happens with this contracts?? As far as I know the have also changed

Reply:
Yes, but is still not done. I put only the ones that I need